department-of-veterans-affairs/caseflow
# Veterans ready for hearing schedule

Connects #7133 

### Description
This PR connects the veterans ready for hearing tab to the backend. It also adds cases to FACOLS so we can test the functionality in demo. 

### Testing Plan
1. Reseed the database
1. Load /hearings/schedule/assign
1. Choose an RO or Central Office
1. Confirm veterans are displayed
